File: pinyougou_parent/pinyougou_seckill_interface/src/main/java/com/pinyougou/seckill/service/SeckillService.java
package com.pinyougou.seckill.service;

import com.pinyougou.pojo.TbSeckillGoods;

import java.util.List;

public interface SeckillService {

    /**
     * 从redis中查询所有的秒杀商品列表
     */
    public List<TbSeckillGoods> findAllSeckillGoodsFromRedis();

    /**
     * 查找指定商品的详情页面
     */
    public  TbSeckillGoods findOneSeckillGoodsFromRedis(Long seckillGoodsId);

    /**
     * 秒杀下单操作
     */
    public void saveSeckillOrder(Long seckillGoodsId,String userId);
}

File: pinyougou_parent/pinyougou_seckill_web/src/main/webapp/js/service/seckillService.js
//服务层
app.service('seckillService',function($http){

    //查询需要秒杀的商品列表
    this.findSeckillList=function(){
        return $http.get('seckill/findSeckillList.do');
    }
    //查询指定的秒杀商品
    this.findOneSeckillGoods=function (seckillGoodsId) {
        return $http.get('seckill/findOneSeckillGoods.do?seckillGoodsId='+seckillGoodsId)
    }
    //秒杀下单
    this.saveSeckillOrder=function (seckillGoodsId) {
        return $http.get('seckill/saveSeckillOrder.do?seckillGoodsId='+seckillGoodsId)
    }
});
